VCFv4.3.tex

```diff
@@ -123,10 +123,10 @@ \subsection{Meta-information lines}
 ##INFO=<ID=ID,Number=number,Type=type,Description="description",Source="description",Version="128">
 \end{verbatim}
 In the above example, the extra fields of ``Source'' and ``Version'' are
-provided. Optional fields should be stored as strings even for numeric values.
+provided. Optional fields must be stored as strings even for numeric values.
 
-It is highly recommended (but not required) that the header
-include tags describing the reference and contigs backing the data contained in
+It is recommended in VCF and required in BCF that the header
```

Comment:
The point of the PR is to remove adjectives as much as possible, following http://www.ietf.org/rfc/rfc2119.txt. This way something will be simply mandatory or recommended/optional, without intermediate states that don't have practical implications.